platform/podjs/src/browserPod.ts

```diff
@@ -43,6 +54,11 @@ class LocalStoragePolyIn implements PolyIn {
         });
     }
 
+    /**
```

Comment:
Please try and generate the documentation. This does not show up, maybe because the class it's in does not have a documentation comment at the beginning.